home/src/main/resources/rdf/abox/filegraph/dynamic-api-individuals-testing.n3

```diff
@@ -23,7 +22,44 @@
         dynapi:hasQueryModel     <https://vivoweb.org/ontology/vitro-dynamic-api/model/full_union> ;
         dynapi:requiresParameter <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/testParam1> ;
         dynapi:sparqlQueryText   "SELECT ?geoLocation ?label\nWHERE\n{\n      ?geoLocation <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://vivoweb.org/ontology/core#GeographicLocation>\n       OPTIONAL { ?geoLocation <http://www.w3.org/2000/01/rdf-schema#label> ?label } \n}\nLIMIT ?limit" .
+        
+#----------------------------------------------------------------------
+#--------------------Added for SolrQuery testing-----------------------
+#----------------------------------------------------------------------
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/solrQuery/genericSolrTextQuery>
+        a                        dynapi:operation, dynapi:solrQuery ;
+        rdfs:label               "Test solr query 1"@en-US ;
+        dynapi:requiresParameter <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/textSearchParam> ;
+        dynapi:solrQueryText   "q=?searchText" .
+    
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/textSearchParam>
+        a                        dynapi:parameter;
+        dynapi:paramName         "searchText";
+        dynapi:hasParameterType  <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/type/XSDstring> ;
+        rdfs:label               "Search text for solr query 1"@en-US . 
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/action/solrTestAction>
```

Comment:
Adding this action is causing the action pool test to fail. There are now 2 actions in the pool after init and 3 after reload action is loaded.

Reply:
Thanks for a quick feedback, will fix today